test/sap-fhir-test-app/webapp/controller/Home.controller.js

```diff
@@ -2,45 +2,56 @@ sap.ui.define([
 	"sap/ui/core/mvc/Controller",
 	"sap-fhir-test-app/utils/Utils",
 	"sap/base/Log"
-], function(Controller, Utils, Log) {
+], function (Controller, Utils, Log) {
 	"use strict";
 
 	return Controller.extend("sap-fhir-test-app.controller.Home", {
-		onInit : function() {
+		onInit: function () {
 			this.initializeRouter();
 		},
 
-		initializeRouter : function() {
+		initializeRouter: function () {
 			this.oRouter = sap.ui.core.UIComponent.getRouterFor(this);
 			this.oRouter.getRoute("home").attachPatternMatched(this.attachModelEventHandlers, this);
+			this.oRouter.getRoute("home").attachPatternMatched(this._refreshBinding, this);
 		},
 
 		/**
 		 * This method shows an use case to register to the events which are provided by the FHIRModel
 		 *
 		 */
-		attachModelEventHandlers : function() {
+		attachModelEventHandlers: function () {
 			if (!this.bInitial) {
 				this.bInitial = true;
 				var oModel = this.getView().getModel();
-				oModel.attachRequestSent(function(oEvent) {
+				oModel.attachRequestSent(function (oEvent) {
 					Log.info("http request send to " + oEvent.getParameter("requestHandle").getUrl() + " with paylod " + oEvent.getParameter("requestHandle").getData());
 				});
-				oModel.attachRequestFailed(function(oEvent) {
-					Log.info("http request failed to " + oEvent.getParameter("requestHandle").getUrl()  + " with paylod " + oEvent.getParameter("requestHandle").getData());
+				oModel.attachRequestFailed(function (oEvent) {
+					Log.info("http request failed to " + oEvent.getParameter("requestHandle").getUrl() + " with paylod " + oEvent.getParameter("requestHandle").getData());
 				});
 			}
 		},
 
-		onNavigateToPatientScreen : function() {
+		_refreshBinding: function () {
+			var oPatientsTile = this.byId("patientTile");
+			if (oPatientsTile) {
+				var oElementBinding = oPatientsTile.getElementBinding();
+				if (oElementBinding) {
+					oElementBinding.refresh();
+				}
+			}
+		},
+
+		onNavigateToPatientScreen: function () {
 			this.oRouter.navTo("patientTable");
 		},
 
-		onNavigateToStructureDefinitionScreen : function() {
+		onNavigateToStructureDefinitionScreen: function () {
 			this.oRouter.navTo("structureDefinitionsOverview");
 		},
 
-		onNavigateToHabibiCancersHistory : function() {
+		onNavigateToHabibiCancersHistory: function () {
```

Comment:
who is habibi?

Reply:
He is a cool guy from San Fransisco Bay area